### DISCUSSION TITLE
🍑 Dream On 🍑

### DISCUSSION THREAD

**Discussion by LivingWithHippos** [LivingWithHippos/unchained-android#228] [Feedback]
### 🏷️ New features:

- [x] custom video players support
- [x] support open web links in app

### 🔎  Enhancements:

- [x] updated libraries
- [x] updated plugins
- [x] swipe between downloads nad torernts tab
- [x] close keyboard after changing search plugins

### 🦟 Bug fixes:

- [x] fix app restore
- [x] fix messages getting lost

#### Notes:

- as usual there are two APKs, a release and a debug one (the bigger one with "debug" in the name), the debug version will allow me to fix bugs more easily (it uses [Countly](https://count.ly/) for telemetry) but if your app does not crash you don' t need to use it
- both releases on GitHub have all the search plugins pre-installed
- it will be uploaded on Google Play a little later after getting feedback from the people using the GitHub version
- Fdroid links will be added when it' s ready

<hr /><em>This discussion was created from the release <a href='https://github.com/LivingWithHippos/unchained-android/releases/tag/4.40.2-beta'>🍑 Dream On 🍑</a>.</em>

**Comment by eazymo** [comment]
Thanks for the update works great!

**Comment by Th3Pi3dP1per** [comment]
Great app. Does everything I need and more. Is there any way to implement searching links already on the real-debrid servers. I'm sure they have an API for it to be feasible. Many thanks

**Reply by LivingWithHippos** [reply]
Hello, sadly it is not possible, the only thing we can do is check if the magnet/torrentis already cached and available but we need the original link (if I've understood correctly what you are asking)
